### PR TITLE
Simplify to just what Ripple needs

### PR DESCRIPTION
**Pull request by michaelcfine**

### CODE REVIEW

File: src/schemas/app-manifest/app/app.json [hunk 1]
```diff
@@ -32,26 +20,14 @@
...
-    },
-    "legacy": {
-      "description": "app.legacy: Legacy app info.",
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/legacy"
-    },
-    "locations": {
-      "description": "app.locations: Allowed, excluded locations.",
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/locations"
-    },
     "signing": {
```
Comment (kpears201) [inline src/schemas/app-manifest/app/app.json:23]: you don't want to remove signing?

File: src/schemas/app-manifest/app/app.json [hunk 2]
```diff
@@ -32,26 +20,14 @@
       "description": "app.runtime: FCNI to execute app, launch params.",
```
Comment (kpears201) [inline src/schemas/app-manifest/app/app.json:20]: I think runtime would be on config.xml right? (They call it "mimetype")

File: src/schemas/app-manifest/distributor/distributor.json [hunk 3]
```diff
@@ -20,10 +20,6 @@
       "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/distributor/fallback",
       "description": "distributor.fallback: Optional settings to control what should happen if the app can't launch."
     },
-    "deviceSkus": {
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/distributor/deviceSkus",
-      "description": "distributor.deviceSkus: Optional allowed and/or excluded Device SKUs to control the subset of devices upon which the app may run."
-    },
     "signing": {
```
Comment (kpears201) [inline src/schemas/app-manifest/distributor/distributor.json:23]: maybe remove signing here too

File: src/schemas/app-manifest/app/app.json (hunk 1 above)
Comment (michaelcfine) [reply src/schemas/app-manifest/app/app.json:23]: Not sure, tbh. Figured I'd leave it in until we have a conversation about it. If you feel strongly, I can remove and we can always add it back.

File: src/schemas/app-manifest/distributor/distributor.json (hunk 3 above)
Comment (michaelcfine) [reply src/schemas/app-manifest/distributor/distributor.json:23]: Not sure, tbh. Figured I'd leave it in until we have a conversation about it. If you feel strongly, I can remove and we can always add it back.

**Review by kpears201 — APPROVED** [review]

**Pull request merged**